cdap-common/bin/common.sh

```diff
@@ -272,6 +273,10 @@ cdap_set_hive_classpath() {
         # tez-site.xml also need to be passed to explore service
         EXPLORE_CONF_FILES=${EXPLORE_CONF_FILES}:${TEZ_CONF_DIR}/tez-site.xml:
       fi
+      if [[ "${HIVE_EXEC_ENGINE}" == "spark" ]]; then
```

Comment:
It would be better to set spark home even if hive engine is not spark. We can choose the engine for each query separately in Hive. Although we don't support it in 3.4, we are planning on adding this support in a later version.